integration/ct_integration_test.go

```diff
@@ -73,25 +73,22 @@ const (
 
 func TestInProcessCTIntegration(t *testing.T) {
 	ctx := context.Background()
-	cfgs := []ct.LogConfig{
+	cfgs := []*ct.LogConfig{
 		{
-			LogID:           6962,
```

Comment:
Long term, createlog.sh and createmap.sh will do RPCs instead of direct storage writes. At that point we'll need to make the scripts return the logID instead of receiving it. For now it should be fine.